pkg/contracts/src/ProxyOwnableUpgrader.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.19;
+
+import {OwnableUpgradeable} from "openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
+
+contract ProxyOwnableUpgrader is OwnableUpgradeable, UUPSUpgradeable {
+    error CallerNotOwner(address _caller, address _owner);
+
+    function initialize(address initialOwner) public initializer {
+        _transferOwnership(initialOwner);
+    }
+
+    function owner() public view override returns (address) {
+        address currentOwner = OwnableUpgradeable.owner();
```

Comment:
call the function as `owner()` instead, that will do external call unecessary, can be internal call

Reply:
Im not sure I can since the virtual base is public